server/src/service/collection.rs

```diff
@@ -0,0 +1,201 @@
+// org.freedesktop.Secret.Collection
+
+use std::collections::HashMap;
+
+use oo7::dbus::api::properties::Properties;
+use oo7::portal::Item;
+use serde::{Deserialize, Serialize, Serializer};
+use zbus::{dbus_interface, zvariant, Error, ObjectServer, SignalContext};
+use zvariant::{ObjectPath, OwnedObjectPath, Signature, Type};
+
+use crate::service::item;
+use crate::KEYRING;
+
+#[derive(Default, Debug, Deserialize)]
```

Comment:
Does this needs to implement Deserialize at all?

Reply:
I guess not :)